libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -539,7 +544,10 @@ void AP_RCProtocol_CRSF::process_link_stats_frame(const void* data)
         }
     }
 
-    _link_status.rf_mode = MIN(link->rf_mode, 7U);
```

Comment:
I am sure @andyp1per put this here for a reason....remember this is used with ELRS also

Reply:
I removed this cap because ELRS now goes up to RFMD 14, it doesn't stop at 7. But if it being higher than 7 will break other stuff in the code, then I can put the cap back on, no problem. Here's the full table:

<img width="695" alt="Screen Shot 2023-12-29 at 19 35 35" src="https://github.com/ArduPilot/ardupilot/assets/9812730/2b9a6a85-73b8-406d-97e8-8f902b734188">

Source: https://www.expresslrs.org/info/signal-health/#rf-mode-indexes-rfmd

Reply:
Ok, in the changes to put the code under the CSRF_EXTENSION wrapper, I have left the rf_mode as is, with the cap at 7.

Reply:
Its fine to change it to 14. Please but in a #define for the constant so that it's clear what is going on. If you change it you need to change the tables as well to be of the larger size - that's why its constrained to stop reading free memory from the tables

Reply:
Ok, thanks for heads up!! Added all the 14 modes, with a define to keep it constrained and also updated the tables. It's all in commit (4c4f167). Let me know if there's anything else I have missed.